webhooks note (#525)
livechat/livechat-public-docs: content/getting-started/guides/webhook-apps/index.mdx

- `token` and `license_id` – your authentication credentials that let you call LiveChat’s Configuration API v2 methods. You won’t need to use them unless you want to make a call to LiveChat’s API right when you receive a webhook. In that case, you just need to pass these `token` and `license_id` credentials in your API call.
- additional information – please read the [Webhook data types](#webhook-data-types) section.

<Warning>

> Note: When your server receives a webhook from LiveChat, it should respond with **HTTP 200** response. Otherwise, LiveChat will retry sending the webhook to your service for a number of times unless it receives the correct **HTTP 200** response.

Reviewer: ```suggestion
When your server receives a webhook from LiveChat, it should respond with **HTTP 200** response. Otherwise, LiveChat will retry sending the webhook to your service a number of times until it receives the correct **HTTP 200** response.
```

Reviewer: - We use the yellow component, so I don't think we need 'Note'.
- 'until' sounds better as we're discussing _time_

> The timeout is set to ~10 seconds and if in that time we won't receive **HTTP 200** response we'll retry sending the webhook up to 10 times within 6 hours.

Reviewer 2: 👏

> The timeout is set to ~10 seconds and if in that time we won't receive **HTTP 200** response we'll retry sending the webhook up to 10 times within 6 hours.

Reviewer: ```suggestion
The timeout is set to ~10 seconds. If we don't receive **HTTP 200** response within that time period,  we'll retry sending the webhook up to 10 times within 6 hours.
```

</Warning>

> If your endpoint is responsible for both receiving data from our webhook and making further actions, like creating record in DB or any other actions, then the whole process may take longer than 10 seconds.

Reviewer: ```suggestion
If your endpoint is responsible for both receiving data from our webhook and making further actions, like creating a record in a database, then the whole process may take longer than 10 seconds.
```

Reviewer: unnecessary repetition of the word 'action'

> In such case we believe that the best solution would be to split your mechanism into 2 separate actions:

Reviewer: ```suggestion
In such a case, we believe,  the best solution would be to split your mechanism into 2 separate actions:
```

> 1) retrieving data from LiveChat webhook (sending http code 200 back to us immediately)

Reviewer: ```suggestion
1) Retrieving data from the LiveChat webhook (sending **HTTP 200** back to us immediately).
```

> 2) further processing this data and passing it to other services

Reviewer: ```suggestion
2) Further processing of the data and passing it to other services.
```

> That would prevent situations in which the process takes longer than our webhook timeout and data would not be pushed again from our side.

Reviewer 2: 👏 👏

> That would prevent situations in which the process takes longer than our webhook timeout and data would not be pushed again from our side.

Reviewer: ```suggestion
That would prevent situations in which the process takes longer than our webhook timeout and data isn't pushed from our side again.
```
